project/Component/NefModels/CarbonModel.swift

```diff
@@ -124,30 +253,56 @@ extension CarbonStyle.Color {
                                                           "yellow": yellow,
                                                           "orange": orange]
     
+    /// Predefined nef `Color`.
     public static let nef = CarbonStyle.Color(r: 140, g: 68, b: 255, a: 1)
+    
+    /// Predefined bow `Color`.
     public static let bow = CarbonStyle.Color(r: 213, g: 64, b: 72, a: 1)
+    
+    /// Predefined white `Color`.
     public static let white = CarbonStyle.Color(r: 255, g: 255, b: 255, a: 1)
+    
+    /// Predefined yellow `Color`.
     public static let yellow = CarbonStyle.Color(r: 255, g: 237, b: 117, a: 1)
+    
+    /// Predefined green `Color`.
     public static let green = CarbonStyle.Color(r: 110, g: 240, b: 167, a: 1)
+    
+    /// Predefined blue `Color`.
     public static let blue = CarbonStyle.Color(r: 66, g: 197, b: 255, a: 1)
+    
+    /// Predefined orange `Color`.
     public static let orange = CarbonStyle.Color(r: 255, g: 159, b: 70, a: 1)
 }
 
-// MARK: Error
+/// Represents an error in a Carbon action.
 public struct CarbonError: Error {
+    
+    /// The snippet of code fails in the action.
     public let snippet: String
+    
+    /// Describe the reason for the failure.
     public let cause: CarbonError.Cause
     
+    /// Initializes a `CarbonError`
+    ///
+    /// - Parameters:
+    ///   - snippet: the snippet of code fails in the action
+    ///   - cause: the reason for the failure.
     public init(snippet: String, cause: CarbonError.Cause) {
         self.snippet = snippet
         self.cause = cause
     }
     
-    // MARK: Error options
+    /// Models the reason of a carbon error.
     public enum Cause: Error, CustomStringConvertible {
+        ///  Could not open Carbon web view with the current configuration.
         case notFound
+        
+        /// Could not take a snapshot.
         case invalidSnapshot
         
+        /// A textual representation of `CarbonError.Cause`.
```

Comment:
I think this does not need to be documented, as its documentation is inherited from `CustomStringConvertible`.

Reply:
yep, you are right - I think so, anyways, I will check it